iamproperty/iamproperty.github.io
# Ds 55/stats

## Summary of Changes:
1. CSS for a new stats class
2. New carousel example to show how the stats can be used

## Rationale
Stats class for typography elements

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /elements/type or /examples/stats-carousel

## Ticket(s)
DS-55

## Notes
